lib/msal-common/src/uCache/serialize/CacheInterface.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Deserializer } from "./Deserializer";
+import { AccessTokenCache, IdTokenCache, RefreshTokenCache, AccountCache, AppMetadataCache, StringDict } from "../../utils/MsalTypes";
+
+export type JSONContent = {
+    accessTokens: StringDict;
+    idTokens: StringDict;
+    refreshTokens: StringDict;
+    accounts: StringDict;
+    appMetadata: StringDict;
+};
+
+export class CacheInterface {
+
+    /**
+     * serialize the JSON blob
+     * @param data
+     */
+    static serializeJSONBlob(data: string): string {
+        return JSON.stringify(data);
+    }
+
+    /**
+     * Parse the JSON blob in memory and deserialize the content
+     * @param cachedJson
+     */
+    static deserializeJSONBlob(jsonFile?: any): JSONContent {
```

Comment:
Seems like this should be `JSONContent`, not `any`?

Reply:
added types and changed this up a bit.